src/cpp/session/modules/NotebookErrors.R

```diff
@@ -13,14 +13,14 @@
 #
 #
 
-.rs.addFunction("handleNotebookError", function() {
-  .rs.recordTraceback(FALSE, function(err) {
-    .Call("rs_recordNotebookError", err)
-  })
-},
-attrs = list(hideFromDebugger = TRUE,
-             errorHandlerType = 4L))
```

Comment:
I think this old integer type here might have been an oversight from a (now very old!) PR: https://github.com/rstudio/rstudio/commit/9e3d350b25fe4ea3165e742b25c5f71cfa1bf9c2.

For example: https://github.com/rstudio/rstudio/commit/9e3d350b25fe4ea3165e742b25c5f71cfa1bf9c2#diff-fcd443acd6f8b9a49ae244cf34eb175ffc1cc65f93f3c6efba80b452397bebf5R217